### PR TITLE
v1.4.0: Additional alerting methods, ping overrides, DSM 7.1 support

### PR DESCRIPTION
**Pull request by beornlake**
- Adds configurable options to give hardware alerts (beeps and lights), as well as an indicator in the filesystem when a VPN loses connection.

### CODE REVIEW

**Review by peledies — APPROVED** [review]
This is a great idea!

I have tested this and it works as intended on my `DS214play`

**Review by peledies — APPROVED** [review]

**Review by peledies — APPROVED** [review]

**Comment by beornlake** [comment]
- Added support for OpenVPN connections in DSM 7.1 (fixes #16).
- Removed support for PPTP VPNs.
- Added override options for `domain_ping` and `gateway_ping` link detection methods. (see https://github.com/ianharrier/synology-scripts/issues/13#issuecomment-952824266)
- Cleaned up document layout and formatting; reorganized some sections.
- Added instructions comment block to the top of the document.
- Bumped to version 1.4.0 (arbitrarily).

File: reconnect-vpn.sh [hunk 1]
```diff
@@ -1,134 +1,231 @@
...
 #===============================================================================
 #         FILE:  reconnect-vpn.sh
 #
 #  DESCRIPTION:  Reconnect a disconnected VPN session on Synology DSM
 #    SOURCE(S):  https://community.synology.com/enu/forum/17/post/53791
 #       README:  https://github.com/ianharrier/synology-scripts
 #
-#       AUTHOR:  Ian Harrier
-#      VERSION:  1.2.1
+#       AUTHOR:  Ian Harrier, Michael Lake
```
Comment (beornlake) [inline reconnect-vpn.sh]:
```suggestion
#       AUTHOR:  Ian Harrier, Michael Lake, Deac Karns
```

**Comment by ianharrier** [comment]
Thank you for contributing! I'd be interested in merging this if you could rebase on the latest version. I've already merged in some of the other features also included here. Let's keep this pull request focused on just adding the new alerting features. Thanks!

**Pull request closed without merging**